components/org.wso2.carbon.siddhi.editor.core/src/main/java/org/wso2/carbon/siddhi/editor/core/internal/EditorMicroservice.java

```diff
@@ -483,7 +489,12 @@ public Response importFile(String path) {
             return Response.status(Response.Status.OK)
                     .entity(content)
                     .type(MediaType.APPLICATION_JSON).build();
-        } catch (IOException e) {
+        } catch (AccessDeniedException e) {
+            Map<String, String> errorMap = new HashMap<>(1);
+            errorMap.put("Error", "File access denied. You don't have enough permission to access");
+            return Response.serverError().entity(errorMap)
+                    .build();
+        }catch (IOException e) {
```

Comment:
formatting issue

Reply:
This code section is already changed in the master branch and this particular code is not there anymore. 
https://github.com/wso2/carbon-analytics/blob/master/components/org.wso2.carbon.siddhi.editor.core/src/main/java/org/wso2/carbon/siddhi/editor/core/internal/EditorMicroservice.java